tests/unit_tests/gen_alpha_poc/test_quaternions.cpp

```diff
@@ -211,6 +211,18 @@ TEST(QuaternionTest, GetRotationVectorFromNullQuaternion) {
     ASSERT_NEAR(v.GetZComponent(), expected.GetZComponent(), 1e-6);
 }
 
+TEST(QuaternionTest, QuaternionToRotationVectorToQuaternion) {
+    Quaternion q(0.707107, 0., 0., 0.707107);
+    auto v = rotation_vector_from_quaternion(q);
+    auto q2 = quaternion_from_rotation_vector(v);
+    Quaternion expected(0.707107, 0., 0., 0.707107);
+
+    ASSERT_NEAR(q2.GetScalarComponent(), expected.GetScalarComponent(), 1e-6);
+    ASSERT_NEAR(q2.GetXComponent(), expected.GetXComponent(), 1e-6);
+    ASSERT_NEAR(q2.GetYComponent(), expected.GetYComponent(), 1e-6);
+    ASSERT_NEAR(q2.GetZComponent(), expected.GetZComponent(), 1e-6);
+}
```

Comment:
This appears to be testing exactly the same thing as the combination of following couple tests.

https://github.com/Exawind/openturbine/blob/0bcd62789576242b6a2a979a8da0908aa94ffe8e/tests/unit_tests/gen_alpha_poc/test_quaternions.cpp#L167C1-L192C1

Reply:
Removed